src/components/FwbFileInput/FwbFileInput.vue

```diff
@@ -15,11 +15,12 @@
     </div>
     <div
       v-else
-      class="flex items-center justify-center"
+      class="flex flex-col items-start justify-center"
       @change="handleChange"
       @dragover="dragOverHandler"
       @drop="dropFileHandler"
     >
+    <span v-if="label !== ''" :class="labelClasses">{{ label }}</span>
```

Comment:
please fix code style according to linter (`npm run lint`)

```
/src/components/FwbFileInput/FwbFileInput.vue
  23:1   warning  Expected indentation of 6 spaces but found 4 spaces   vue/html-indent
  23:31  warning  ':class' should be on a new line                      vue/max-attributes-per-line
```

Reply:
I did npm run format.  Lint was giving too many errors.